docs/examples/voting.rst

```diff
@@ -52,14 +52,14 @@ of votes.
             uint voteCount; // number of accumulated votes
         }
 
-        address public chairperson;
+        address external chairperson;
```

Comment:
For state variables, it should be public. Only for functions, we need to change it.

Reply:
ok i will change it for the functions only

Reply:
BTW, `external` for state variables wouldn't compile.